### DISCUSSION TITLE
V5

### DISCUSSION THREAD

**Discussion by aureochocob0n** [Crownicles/Crownicles#2574] [Suggestions fermées]
Bonjour, je propose un reset de points avec la venue de la v5, ainsi le classement représentera vraiment ceux qui sont actifs et encouragera les joueurs à jouer

**Comment by BastLast** [comment]
Je propose de faire un sondage de qui veux reset et qui veux pas etre reset et je reset que ceux qui ne veulent pas l'être

**Comment by Ravenclaw-l** [comment]
Je ne sais pas laquelle est pire : la suggestion d'Aureo ou celle de Draft.

**Reply by StrangerIM** [reply]
Cette discussion me fait peur

**Comment by BastLast** [comment]
Ou alors je reset que raven et stranger

**Comment by kirit000** [comment]
Je veux bien être reset alors ne me reset pas

**Comment by Ravenclaw-l** [comment]
Si mes points étaient supprimés comme le suggère Aureo, je ne vois aucun intérêt à continuer à jouer. Jeter deux ans de jeu à la poubelle, why ?

**Comment by BastLast** [comment]
Bon je propose un truc je reset tous ceux qui n'écrivent pas chocolat en réponse à ce commentaire
